.buildkite/commands/run-ui-tests.sh

```diff
@@ -19,8 +19,8 @@ tar -xf build-products-jetpack.tar
 echo "--- :rubygems: Setting up Gems"
 install_gems
 
-echo "--- :cocoapods: Setting up Pods"
-install_cocoapods
```

Comment:
We don't need the pods source code because we're not actually building the app.

Unfortunately, we still need the Swift packages because of how Fastlane uses `xcodebuild` under the hood (I think)